cl/search/management/commands/generate_cap_crosswalk.py

```diff
@@ -74,6 +81,25 @@ def handle(self, *args: Any, **options: Any) -> None:
         self.single_volume = options["volume"]
         self.output_dir = options["output_dir"]
         self.start_from_reporter = options["start_from_reporter"]
+        self.updated_after = None
+        if options["updated_after"]:
+            try:
+                self.updated_after = datetime.fromisoformat(
+                    options["updated_after"]
+                )
+            except ValueError:
+                try:
+                    self.updated_after = datetime.strptime(
```

Comment:
maybe you can follow the same style and do this:

`self.updated_after = options["updated_after"]`

I think you can do this in one line, if something fails it will raise the ValueError exception:

`self.updated_after = datetime.strptime(updated_after, "%Y-%m-%d").replace(tzinfo=pytz.UTC)
`